cmd/maintenance.go

```diff
@@ -18,10 +18,10 @@ func (d DeisCmd) MaintenanceInfo(appID string) error {
 		return err
 	}
 
-	if *appSettings.Maintenance {
-		d.Println("Maintenance mode is on.")
+	if appSettings.Maintenance == nil || !*appSettings.Maintenance {
+		d.Println("Maintenance mode is disabled.")
```

Comment:
i kept it as on/off as heroku has it that way.https://devcenter.heroku.com/articles/maintenance-mode